### PR TITLE
pythonPackages.tqdm: 4.54.1 -> 4.58.0

### PR DESCRIPTION
**Pull request by sternenseemann**
###### Motivation for this change

afdko 3.6.1 depends on tqdm >= 4.58

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/tqdm/default.nix [hunk 1]
```diff
@@ -7,22 +7,32 @@
...
   nativeBuildInputs = [
     setuptools_scm
   ];
 
-  checkInputs = [ nose coverage glibcLocales flake8 pytestCheckHook ];
+  checkInputs = [
+    nose
+    coverage
+    glibcLocales
+    flake8
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/tqdm/default.nix:31]:
```suggestion
    coverage
```

Are the removed checkInputs no longer required?

Comment (sternenseemann) [reply pkgs/development/python-modules/tqdm/default.nix:31]: No `checkInputs` where removed, just flowed them into multiple lines as the line was getting pretty long

Comment (SuperSandro2000) [reply pkgs/development/python-modules/tqdm/default.nix:31]: I meant to ask if all of them are still required because upstream did not list them as requirement.

Comment (sternenseemann) [reply pkgs/development/python-modules/tqdm/default.nix]: Sorry, misunderstood you there. You are right, those inputs can be removed. I've also discovered some optional tests which I've also enabled (don't seem particularly expensive).

**Comment by sternenseemann** [comment]
```
https://github.com/NixOS/nixpkgs/pull/114500                                          
6 packages marked as broken and skipped:                                                                                                                                    
python37Packages.optuna python37Packages.pymc3 python37Packages.skorch python38Packages.optuna python38Packages.pymc3 python38Packages.skorch

6 packages failed to build:                                                                                                                                                 
apache-airflow python37Packages.pytorch-metric-learning python37Packages.shap python37Packages.textacy python38Packages.shap python38Packages.textacy

164 packages built:                                                                                                                                                         
aiodnsbrute backblaze-b2 check_systemd checkov conan deja-dup duplicity duply evdevremapkeys evillimiter ffmpeg-normalize gdown gnome-keysign jellyfin-mpv-shim kaggle keepk
ey_agent ldeep ledger_agent lieer luigi magic-wormhole manim mathlibtools mnemosyne noto-fonts-emoji nrfutil ocrmypdf papis plex-mpv-shim python37Packages.afdko python37Pac
kages.ansible-kernel python37Packages.ansible-runner python37Packages.b2sdk python37Packages.baselines python37Packages.colorcet python37Packages.datasets python37Packages.
datashader python37Packages.dipy python37Packages.dnachisel python37Packages.gdown python37Packages.genome-collector python37Packages.gradient python37Packages.gradient-uti
ls python37Packages.graphtage python37Packages.gtts python37Packages.holoviews python37Packages.hvplot python37Packages.hyperopt python37Packages.ignite python37Packages.in
take python37Packages.internetarchive python37Packages.kaggle python37Packages.keepkey_agent python37Packages.labelbox python37Packages.ledger_agent python37Packages.libage
nt python37Packages.lsassy python37Packages.magic-wormhole python37Packages.mathlibtools python37Packages.mesa python37Packages.moviepy python37Packages.msldap python37Pack
ages.nagiosplugin python37Packages.nbsmoke python37Packages.ndjson python37Packages.nltk python37Packages.nototools python37Packages.panel python37Packages.papermill python
37Packages.papis python37Packages.parfive python37Packages.perfplot python37Packages.plexapi python37Packages.proglog python37Packages.pyppeteer python37Packages.pypykatz p
ython37Packages.pyro-ppl python37Packages.python-daemon python37Packages.python-miio python37Packages.python-mpv-jsonipc python37Packages.pytorch-lightning python37Packages
.pywick python37Packages.rbtools python37Packages.rising python37Packages.runway-python python37Packages.sacremoses python37Packages.salmon-mail python37Packages.spacy pyth
on37Packages.sunpy python37Packages.thinc python37Packages.toggl-cli python37Packages.tqdm python37Packages.transformers python37Packages.trezor_agent python37Packages.tube
up python37Packages.twine python38Packages.afdko python38Packages.ansible-kernel python38Packages.ansible-runner python38Packages.b2sdk python38Packages.baselines python38P
ackages.colorcet python38Packages.datasets python38Packages.datashader python38Packages.dipy python38Packages.dnachisel python38Packages.genome-collector python38Packages.g
radient python38Packages.gradient-utils python38Packages.graphtage python38Packages.gtts python38Packages.holoviews python38Packages.hvplot python38Packages.hyperopt python
38Packages.ignite python38Packages.intake python38Packages.internetarchive python38Packages.labelbox python38Packages.libagent python38Packages.lsassy python38Packages.mesa
 python38Packages.moviepy python38Packages.msldap python38Packages.nagiosplugin python38Packages.nbsmoke python38Packages.ndjson python38Packages.nltk python38Packages.noto
tools python38Packages.panel python38Packages.papermill python38Packages.parfive python38Packages.perfplot python38Packages.plexapi python38Packages.proglog python38Package
s.pyppeteer python38Packages.pypykatz python38Packages.pyro-ppl python38Packages.python-daemon python38Packages.python-miio python38Packages.python-mpv-jsonipc python38Pack
ages.pytorch-lightning python38Packages.pytorch-metric-learning python38Packages.pywick python38Packages.rbtools python38Packages.rising python38Packages.runway-python pyth
on38Packages.sacremoses python38Packages.salmon-mail python38Packages.spacy python38Packages.sunpy python38Packages.thinc python38Packages.toggl-cli python38Packages.tqdm p
ython38Packages.transformers trezor_agent python38Packages.tubeup python38Packages.twine r128gain tartube theharvester tinyprog tts twitter-color-emoji urlwatch
```

**Comment by sternenseemann** [comment]
* Can't reproduce the failure of `apache-airflow` anymore
* broken on `master` as well:
  * `pytorch-metric-learning`
  * `shap`
  * `textacy`

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).
If you have any questions or problems please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 114500` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
3 packages built:
</summary>
<ul>
<li>python37Packages.tqdm</li>
<li>python38Packages.tqdm</li>
<li>python39Packages.tqdm</li>
</ul>
</details>
